app/src/main/java/de/tum/in/tumcampusapp/component/other/wifimeasurement/WifiMeasurementManager.java

```diff
@@ -1,94 +0,0 @@
-package de.tum.in.tumcampusapp.component.other.wifimeasurement;
-
-import android.content.Context;
-import androidx.annotation.NonNull;
-
-import java.io.IOException;
-import java.util.List;
-
-import de.tum.in.tumcampusapp.api.app.TUMCabeClient;
-import de.tum.in.tumcampusapp.api.app.model.TUMCabeStatus;
-import de.tum.in.tumcampusapp.component.other.wifimeasurement.model.WifiMeasurement;
-import de.tum.in.tumcampusapp.database.TcaDb;
-import de.tum.in.tumcampusapp.utils.Utils;
-import retrofit2.Call;
-import retrofit2.Callback;
-import retrofit2.Response;
-
-public class WifiMeasurementManager {
-    public static final String WIFI_SCAN_MINIMUM_BATTERY_LEVEL = "WIFI_SCAN_MINIMUM_BATTERY_LEVEL";
-
-    private final WifiMeasurementDao dao;
```

Comment:
What about the `WifiMeasurementDao` and `WifiMeasurement` those should probably also be removed?

Reply:
Good catch!
We probably also need a mirgation to drop that table